BridgeCareApp/VuejsApp/src/components/DetailedReport.vue

```diff
@@ -174,9 +191,9 @@
          */
         onRunSimulation() {
```

Comment:
Get rid of this message.

Reply:
Do we not want any modal popup before running the simulation?

Reply:
No

Reply:
As discussed, we are keeping the popup for now